### PR TITLE
Update howto doc on Channel in chrome extension; various edits for cl…

### PR DESCRIPTION
**Pull request by geoffmenegay**
…arity.

### CODE REVIEW

File: docs/extensionuse.rst [hunk 1]
```diff
@@ -8,26 +8,33 @@ Navigate to any merchant that sells the product you want to SmartLink and click
...
+Step 3. Associate the product link with the story name
 ------------------------------------------------------
 
 You can do this in one of two ways:
 
 **1. Create a new article link**
 
-  - Type article name and date next to the gold “+” button, then click SMARTLINK to create a link.
-  - (Note: Match the article title verbatim to your site, e.g., “The 15 Best Eye Creams 8/29/17”)
+  - Type the article name and date next to the gold “+” button, then click CREATE + LINK to create the link and the article.
```
Comment (myfashionhub) [inline docs/extensionuse.rst]: Avoid smart quotes

File: docs/extensionuse.rst [hunk 2]
```diff
@@ -38,12 +45,12 @@ Need to ensure the link only goes to one destination?
 .. image:: _static/extension_instructions/Dynamic.png
 .. image:: _static/extension_instructions/Exclusive.png
 
-Step 4. Create your SmartLink
+Step 5. Create your SmartLink
 -----------------------------
 
-  - Once you have clicked the blue SMARTLINK button, the link will automatically copy to your clipboard, and is ready for use in your content.
+  - Once you have created the SmartLink, its URL will be copied to your clipboard aautomatically and is ready for use.
```
Comment (csdev) [inline docs/extensionuse.rst]: typo "aautomatically"

Comment (geoffmenegay) [reply docs/extensionuse.rst]: fixed

File: docs/extensionuse.rst (hunk 1 above)
Comment (geoffmenegay) [reply docs/extensionuse.rst]: replaced all 3 sets

**Pull request merged**